app/javascript/components/pages/CityNew.js

```diff
@@ -1,11 +1,265 @@
-import React from "react";
-
-const CityNew=  () => {
-    return (
-        <>
-        <h3> This is City New/h3</h3>
-        </>
-    )
+import React, { useState } from "react"
+import { Button, Form, FormGroup, Label, Input, FormText } from "reactstrap"
+
+const CityNew = ({ createCity }) => {
+  // I set the default state for the options that I used a drop select for. These can still be updated by the user and submitted correctly, but if the user does NOT change the drop down, it will default to the value that's shown on the page. (This will be the first value in the options.)
+  const [newCity, setNewCity] = useState({
+    city_name: "",
+    country_name: "SELECT COUNTRY",
+    notes: "",
+  })
+
+  const handleChange = (e) => {
+    setNewCity({ ...newCity, [e.target.name]: e.target.value })
+    console.log(e.target.name, e.target.value)
+  }
+
+  const handleSubmit = () => {
+    console.log("User Submitted Information:", newCity)
+  }
+
+  return (
+    <>
+      <div className="form">
+        <div className="form-card">
+          <Form>
+            <FormGroup>
+              <Label for="city">City</Label>
+              <Input
+                type="text"
+                name="City"
```

Comment:
I think assigning this id might pose. Oftentimes it is removed from the Form. I would say consider removing if any issues arise. Looks like the value for the id and name were capitalize vice being written in lower case.